<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="addBtn">
           <property name="text">
            <string>add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="showBtn">
           <property name="text">
            <string>show</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="modBtn">
           <property name="text">
            <string>mod</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteBtn">
           <property name="text">
            <string>delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="findBtn">
           <property name="text">
            <string>find</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sortBtn">
           <property name="text">
            <string>sort</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cleanBtn">
           <property name="text">
            <string>clean</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="mainTableWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
